### DISCUSSION TITLE
Dependency dashboard workflow

### DISCUSSION THREAD

**Discussion by micah** [renovatebot/renovate#21226] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

I'm curious to know how people integrate the dashboard into your workflow. In gitlab, many people use labels for moving issues forwards on a board (eg. Next, Doing, Done), and when new issues come in you typically triage them by placing labels on them...but a persistent issue that never gets closed doesn't fit into this workflow, so I'm curious how people manage it.

Also, how do you handle empty dashboards? I have a repository that renovate on-boarded and it immediately created a dependency dashboard issue, even though there were no updates to the dependencies. If I close that issue, it will just be re-created next time renovate runs.  This isn't particularly useful, so I'd probably disable the dashboard here, but then I will never get it in the future.

It would be nice if the dashboard only appeared if there are actual issues to track.

**Comment by HonkingGoose** [comment]
> It would be nice if the dashboard only appeared if there are actual issues to track.

Wish granted! 🧞 Set `dependencyDashboardAutoclose` [^docs] to `true`. Now the Dependency Dashboard only appears when Renovate has  PRs for you. 😉 

[^docs]: [Docs for dependencyDashboardAutoclose](https://docs.renovatebot.com/configuration-options/#dependencydashboardautoclose)

**Comment by HonkingGoose** [comment]
> In gitlab, many people use labels for moving issues forwards on a board (eg. Next, Doing, Done), and when new issues come in you typically triage them by placing labels on them...but a persistent issue that never gets closed doesn't fit into this workflow, so I'm curious how people manage it.

I'd use the "Next" label for the Dependency Dashboard. This is because the Dependency Dashboard shows thing that need to be done like:

- "open update PRs"
- "pending creating update PRs"
- updates you've deliberately ignored but probably should update at a later time

You can use the `dependencyDashboardLabels` [^docs] config option to tell Renovate bot what label to use for the Dependency Dashboard issue.

[^docs]: [Renovate docs for `dependencyDashboardLabels`](https://docs.renovatebot.com/configuration-options/#dependencydashboardlabels)
